akka-actor/src/main/resources/reference.conf

```diff
@@ -350,6 +350,19 @@ akka {
         fallback = "fork-join-executor"
       }
 
+      affinity-pool-executor {
+        # The number of threads that this executor will maintain in its pool
+        num-threads = 8
```

Comment:
If we go that way, we will sacrifice some of the actor to thread affinity that might have been in place at a certain point. Example of that is. If we have 5 workers (threads) and Actor A gets executed on thread 3 and the next time it gets scheduled we have 9 threads because the pool grew, it might execute on thread num 7.